src/internal/minikubetestenv/client_factory.go

```diff
@@ -223,3 +231,38 @@ func AcquireCluster(t testing.TB, opts ...Option) (*client.APIClient, string) {
 	deleteAll(t, mc.client)
 	return mc.client, assigned
 }
+
+// set the coredns ttl to 0 to cut down on waiting time when tearing down and creating new pachd instances.
+func coreDNSConfigMap(t testing.TB) {
```

Comment:
I think this belongs in deploy.go. We try containing the k8s / infrastructure specific details there if possible